WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -1,11 +1,13 @@
+// swiftlint:disable next opening_brace
```

Comment:
This makes Lint ignore the rule for the whole file, which is not something desired. What do you think of this?

```swift
                // swiftlint:disable next opening_brace
                { $0.buttons["product-add-button"] },
                { $0.buttons["product-scan-button"] },
                { $0.buttons["product-search-button"] }
                // swiftlint:enable next opening_brace
```

Reply:
I was contemplating between the two and decided to go with the one-liner 😅 . No strong opinions here, I'll make the change to have it disabled only for that section.